host/workflowidratelimit_test.go

```diff
@@ -57,12 +129,14 @@ func (s *IntegrationSuite) TestWorkflowIDSpecificRateLimits() {
 
 	// The ratelimit is 5 per second, so we should be able to start 5 workflows without any error
 	for i := 0; i < 5; i++ {
+		s.Require().NotNil(s.engine)
```

Comment:
Having the check in single every loop make me think `s.engine` is assigned during the execution.
I think it should be enough to check outside as a pre-requisite. Maybe even in the `SetupSuite` where it is assigned?

Reply:
Yeah good catch, will delete, it's a debug leftover